### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by hanhou** [AllenNeuralDynamics/aind-disrnn-pipeline#9]
A place to keep Han's roadmap for disRNN.

###  disRNN on nonstationary behavior
Animal behavior is nonstationary, which limits the reliability of session-wise cross-validation when fitting disRNN models. I plan to run in silico experiments to study the effects of nonstationarity on disRNN performance and explore potential mitigation strategies.

- [x] Across-session nonstationarity
    - [x] Quantify its impact on disRNN fitting
        - [x] https://github.com/AllenNeuralDynamics/aind-disrnn-wrapper/issues/7
        - [x] Introduce session-wise variability in model params (bias, learning rate, softmax temp, etc.) and assess disRNN performance https://github.com/AllenNeuralDynamics/aind-disrnn-pipeline/issues/10
        - [x] Add baselines 
          - [x] the ground truth likelihood per trial https://github.com/AllenNeuralDynamics/aind-disrnn-wrapper/issues/30
          - [x] the RL fitted LPT https://github.com/AllenNeuralDynamics/aind-disrnn-pipeline/issues/12
- [ ] Session embedding #13 
- [ ] Subject embedding #14
- [ ] Within-session nonstationarity #15
    - [ ] Discrete states within a session
    - [ ] Drifting params within a session
    
### Shared infrastructure
- [x] https://github.com/AllenNeuralDynamics/aind-disrnn-wrapper/pull/15
- [ ]  Visualize bottleneck sigma and m over steps
- [ ]  ~~Early stopping mechanism?~~

**Comment by hanhou** [comment]
<img width="1132" height="570" alt="Image" src="https://github.com/user-attachments/assets/79fe55c0-37dd-4373-80f0-4e3d79d0d51e" />
